### ISSUE TITLE
Max length for splitStrings

### ISSUE THREAD

**Issue by punkfest** [javascript-obfuscator/javascript-obfuscator#1068]
**Is your feature request related to a problem? Please describe.**
_I'm always frustrated when_ I have a very long string that is split in very small chunks by `splitStringsChunkLength`, eg:
`data:image/gif;base64,R0lGODdhEAAQAMwAAPj7+FmhUYjNfGuxYYDJdYTIeanOpT+DOTuANXi/bGOrWj6CONzv2sPjv2CmV1unU4zPgISg6DJnJ3ImTh8Mtbs00aNP1CZSGy0YqLEn47RgXW8amasW7XWsmmvX2iuXiwAAAAAEAAQAAAFVyAgjmRpnihqGCkpDQPbGkNUOFk6DZqgHCNGg2T4QAQBoIiRSAwBE4VA4FACKgkB5NGReASFZEmxsQ0whPDi9BiACYQAInXhwOUtgCUQoORFCGt/g4QAIQA7`

splitString is great to hide values like `document` (`"docu"+"ment"`), but performs horribly with absurdly long strings.

**Describe the solution you'd like**
I would like to specify a maximum string length. Past this threshold, strings are ignored and not split in tiny chunks.

**Describe alternatives you've considered**
Disabling splitString, and crying. They're mediocre alternatives, I don't recommend them

**Comment by BlackYuzia** [comment]
What do you mean? 
Split with too long strings have too slow performance?

**Comment by punkfest** [comment]
Yes, this leads to slow performances -- but more importantly, the generated script can be 2x to 3x as big due to it.
